tests/periph_uart/tests/02__periph_uart_mode.robot

```diff
@@ -17,8 +18,7 @@ Force Tags          periph  uart
 *** Test Cases ***
 Even Parity 8 Bits
     DUT UART mode should exist  dev=1
-    PHILIP.Setup Uart		parity=${UART_PARITY_EVEN}
-    API Call Should Succeed     Uart Init
```

Comment:
`UART init` is not required?

Reply:
UART init is required.  Actually it should be called before any mode change which is why I include it in with the uart mode call.  In the current representation it was not called before the second mode change.